index.html

```diff
@@ -26,16 +26,19 @@
                     <div class="intro-text">
                         <span class="brand">Greetings, Java Hipster!</span>
                         <hr class="star-light">
-                        <span class="open-collective-info">
-                            JHipster is now on <a href="https://opencollective.com/generator-jhipster" target="_blank" rel="noopener">Open Collective</a>,
+                        <p class="project-description">
+                            JHipster is a development platform to quickly generate, develop, & deploy modern web applications & microservice architectures.
```

Comment:
"Web" instead of "web"?

Reply:
I've learned through all my editors when writing that it's better to not capitalize words. It's kinda like internet, which is [not supposed to be capitalized any more](https://www.wired.com/2015/10/should-you-be-capitalizing-the-word-internet/).

Reply:
Oh I've learned something today, thank you!